runtime/master/src/main/java/org/apache/nemo/runtime/master/metric/MetricStore.java

```diff
@@ -278,6 +278,10 @@ public void saveOptimizationMetricsToDB(final String address, final String jobId
                                           final String dbId, final String dbPasswd) {
     final String[] syntax = {"SERIAL PRIMARY KEY"};
 
+    if (MetricUtils.metaDataLoaded()) {
+      MetricUtils.loadMetaData();
+    }
```

Comment:
Why does it call `loadMetaData()` again when metadata is already loaded?

Is the previously loaded metadata stable?
Then please write a comment.